src/mantarray_waveform_analysis/peak_detection.py

```diff
@@ -743,18 +797,20 @@ def calculate_area_under_curve(  # pylint:disable=too-many-locals # Eli (9/1/20)
             rising_coords,
             falling_coords,
         )
-        auc_per_twitch.append(int(round(auc_total, 0)))
+        if round_to_int:
+            auc_total = int(round(auc_total, 0))
+        auc_per_twitch.append(auc_total)
 
-    return np.asarray(auc_per_twitch, dtype=np.int64)
+    return np.asarray(auc_per_twitch, dtype=float)
 
 
 def _calculate_trapezoid_area(
     left_x: int,
     right_x: int,
     left_y: int,
     right_y: int,
-    rising_coords: Tuple[int, int],
-    falling_coords: Tuple[int, int],
+    rising_coords: Tuple[Union[float, int], Union[float, int]],
```

Comment:
minor note for future: the time value (x coordinate) will still always be an integer, right?